docs/team/njxue.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Ng Jing Xue's Project Portfolio Page
+---
+
+### Project: TA Assist
+
+TA Assist is a desktop application targeted at NUS Teaching Assistants (TA). It helps them to keep track of their students' grades, attendance, and work submission status of relevant modules.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
+
+Given below are my contributions to the project.
+
+* **New Feature**: To be added soon
+
+* **Code contributed**: To be added soon
```

Comment:
Perhaps can put in the RepoSense link at this stage? It's available under https://nus-cs2103-ay2223s1.github.io/website/admin/teamList.html.

Reply:
Oh right, thanks for reminding!